build-docker-images/wazuh-indexer/config/config.sh

```diff
@@ -104,3 +104,6 @@ cp -pr /wazuh-certificates/root-ca.key ${TARGET_DIR}${CONFIG_DIR}/root-ca.key
 cp -pr /wazuh-certificates/root-ca.pem ${TARGET_DIR}${CONFIG_DIR}/root-ca.pem
 cp -pr /wazuh-certificates/admin.pem ${TARGET_DIR}${CONFIG_DIR}/admin.pem
 cp -pr /wazuh-certificates/admin-key.pem ${TARGET_DIR}${CONFIG_DIR}/admin-key.pem
+
+chmod -R 500 ${TARGET_DIR}${CONFIG_DIR}
+chmod -R 400 ${TARGET_DIR}${CONFIG_DIR}/*
```

Comment:
Why the files aren't inside certs folder?

Reply:
Done [d927698](https://github.com/wazuh/wazuh-docker/pull/604/commits/d9276985c3ea8165610b749cf00f29ee44609757)